### PR TITLE
Move off of the old arm32 queue

### PR DESCRIPTION
**Pull request by jashook**
/cc @mmitche 

/cc @trylek @lpereira @ilyas1974

### CODE REVIEW

File: eng/platform-matrix.yml [hunk 1]
```diff
@@ -36,11 +36,11 @@ jobs:
...
       helixQueues:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - (Ubuntu.1804.Arm32.Open)Ubuntu.1604.Arm32.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-30f6673-20190814153226
+        - (Ubuntu.1804.Arm32.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-30f6673-20190814153226
       - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-        - (Debian.9.Arm32)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-74c9941-20190620155841
-        - (Ubuntu.1604.Arm32)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm32v7-a45aeeb-20190620160312
-        - (Ubuntu.1804.Arm32)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-09a60ed-20190620155854
+        - (Debian.9.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-74c9941-20190620155841
+        - (Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm32v7-a45aeeb-20190620160312
```
Comment (lpereira) [inline eng/platform-matrix.yml:42]: This says "Ubuntu 18.04", but then references "ubuntu-16.04" later in the line. Is this intended?

Comment (hoyosjs) [reply eng/platform-matrix.yml:42]: Yes, the arm host machine is a Ubuntu ARM 18.04 machine running the cross-build container that runs on 16.04. We use the same test container on top of the new queue of agents. It looks like we want to use new hosts and not new test images.

Comment (jashook) [reply eng/platform-matrix.yml:42]: Correct this is by design.

**Comment by jashook** [comment]
Change is good, looking for a review to merge.

**Review by hoyosjs — APPROVED** [review]
Checked the queues

**Pull request merged**